scripts/sct_straighten_spinalcord.py

```diff
@@ -240,6 +240,6 @@ def main(args=None):
 # START PROGRAM
 # ==========================================================================================
 if __name__ == "__main__":
-    sct.init_sct()
+    sct.init_sct(log_level='DEBUG')
```

Comment:
@zougloub i'm not sure how to elegantly catch the parser argument at this level (because it is supposed to be caught inside the main). 

Should I parse here instead? Or should we move init_sct() inside the main()?